hx_requests/hx_requests.py

```diff
@@ -231,12 +231,11 @@ def _render_templates(self, templates, blocks, **kwargs) -> str:
         # If blocks is a dict then we are using multiple blocks with multiple templates
         if isinstance(blocks, dict):
             for template, block in blocks.items():
-                html += render_with_context(template, block)
-            if templates and isinstance(templates, str):
-                html += render_with_context(templates, None)
-            elif isinstance(templates, list):
-                for template in templates:
-                    html += render_with_context(template, None)
```

Comment:
This code is needed for rendering additional templates (without blocks) provided in `POST_template`